src/index.html

```diff
@@ -11,7 +11,168 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+    onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    >
```

Comment:
Attributes here should have 2 additional spaces.
```suggestion
    <form
      onsubmit="onSubmit()"
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
    >
```